test/spec/flink_validation_fhir_spec.rb

```diff
@@ -9,35 +9,35 @@
     @flink_api_oauth_token = @appid_helper.get_access_token('hri_integration_tenant_test_data_integrator', '', ENV['APPID_FLINK_AUDIENCE'])
     @hri_oauth_token = @appid_helper.get_access_token('hri_integration_tenant_test_data_integrator', 'tenant_test hri_data_integrator', ENV['APPID_HRI_AUDIENCE'])
     @mgmt_api_helper = HRITestHelpers::MgmtAPIHelper.new(ENV['HRI_INGRESS_URL'], @iam_token)
     @elastic = HRITestHelpers::ElasticHelper.new({url: ENV['ELASTIC_URL'], username: ENV['ELASTIC_USER'], password: ENV['ELASTIC_PASSWORD']})
     @record_validator = KafkaRecordValidator.new
 
     timestamp = Time.now.to_i
-    @input_topic = ENV['INPUT_TOPIC'].gsub('.in', "-#{@branch_name}-#{timestamp}.in")
-    @output_topic = ENV['OUTPUT_TOPIC'].gsub('.out', "-#{@branch_name}-#{timestamp}.out")
-    @notification_topic = ENV['NOTIFICATION_TOPIC'].gsub('.notification', "-#{@branch_name}-#{timestamp}.notification")
-    @invalid_topic = ENV['INVALID_TOPIC'].gsub('.invalid', "-#{@branch_name}-#{timestamp}.invalid")
+    @input_topic = ENV['INPUT_TOPIC'].gsub('.in', "-#{@git_branch}-#{timestamp}.in")
+    @output_topic = ENV['OUTPUT_TOPIC'].gsub('.out', "-#{@git_branch}-#{timestamp}.out")
+    @notification_topic = ENV['NOTIFICATION_TOPIC'].gsub('.notification', "-#{@git_branch}-#{timestamp}.notification")
+    @invalid_topic = ENV['INVALID_TOPIC'].gsub('.invalid', "-#{@git_branch}-#{timestamp}.invalid")
     @event_streams_helper.create_topic(@input_topic, 1)
     @event_streams_helper.create_topic(@output_topic, 1)
     @event_streams_helper.create_topic(@notification_topic, 1)
     @event_streams_helper.create_topic(@invalid_topic, 1)
     @event_streams_helper.verify_topic_creation([@input_topic, @output_topic, @notification_topic, @invalid_topic])
 
-    @output_consumer_group = "hri-flink-validation-fhir-#{@branch_name}-#{timestamp}-output-consumer"
-    @notification_consumer_group = "hri-flink-validation-fhir-#{@branch_name}-#{timestamp}-notification-consumer"
-    @invalid_consumer_group = "hri-flink-validation-fhir-#{@branch_name}-#{timestamp}-invalid-consumer"
-    @kafka = Kafka.new(ENV['KAFKA_BROKERS'], client_id: "hri-flink-validation-fhir-#{@branch_name}-#{timestamp}", connect_timeout: 10, socket_timeout: 10, sasl_plain_username: 'token', sasl_plain_password: ENV['SASL_PLAIN_PASSWORD'], ssl_ca_certs_from_system: true)
+    @output_consumer_group = "flink-validation-fhir-#{@git_branch}-#{timestamp}-output-consumer"
+    @notification_consumer_group = "flink-validation-fhir-#{@git_branch}-#{timestamp}-notification-consumer"
+    @invalid_consumer_group = "flink-validation-fhir-#{@git_branch}-#{timestamp}-invalid-consumer"
+    @kafka = Kafka.new(ENV['KAFKA_BROKERS'], client_id: "flink-validation-fhir-#{@git_branch}-#{timestamp}", connect_timeout: 10, socket_timeout: 10, sasl_plain_username: 'token', sasl_plain_password: ENV['SASL_PLAIN_PASSWORD'], ssl_ca_certs_from_system: true)
```

Comment:
You're removing the `hri-` prefix here which makes me think the changes may be in reverse here too.  I would double check all the changes to the `.rb` files.

Reply:
I think that @FRicci618 just forgot to replace the instances of `flink-validation-fhir` with `hri-flink-validation-fhir` in 3 of the .rb files.

I have now replaced all of the instances of `flink-validation-fhir` with `hri-flink-validation-fhir` in this .rb file.

I checked several of the files, against the IBM Repo version of the develop branch and I think we got the correct set of changes here.